website/facedetection/models.py

```diff
@@ -0,0 +1,332 @@
+import os
+from secrets import token_urlsafe
+
+from django.db import models
+from django.db.models import Count, IntegerField, Value
+from django.db.models.functions import Coalesce
+
+from queryable_properties.managers import QueryablePropertiesManager
+from queryable_properties.properties import AnnotationProperty
+from thumbnails.fields import ImageField
+
+from members.models import Member
+from photos.models import Photo
+
+
+class FaceDetectionUser(Member):
+    class Meta:
+        proxy = True
+
+
+def authentication_token() -> str:
```

Comment:
Technically, this is a authorization token and not an authentication token

Reply:
Hmm depends on how it's used though. changed it to secure_token